### ISSUE TITLE
GoLint support for entire package?

### ISSUE THREAD

**Issue by ivulfson** [fatih/vim-go#461]
Looks like GoLint only supports running on the current file.  Would it be possible to add support for running golint on the entire package, when editing one of the package files?

**Comment by fatih** [comment]
Hi @ivulfson.

`golint` supports packages: https://github.com/golang/lint#usage so yes it's possible. Thanks for bringing up the idea. I'll mark this as an improvement and I'm going to look at it once I have proper time.

**Comment by fatih** [comment]
This is now supported with the merged #493 PR. @ivulfson Please test it and report us any problems you have :)

**Comment by ivulfson** [comment]
Confirmed working, after I changed my .vimrc thusly:

```
-au FileType go nmap <leader>l  :GoLint<CR>
+au FileType go nmap <leader>l  :GoLint .<CR>
```

That only became apparent after looking at the code diff. :)  Not sure if that's what I'm supposed to do, though.
